subsys/bluetooth/mesh/brg_cfg.c

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Implimentation for Bridging Table state of Subnet Bridge feature
+ * in Bluetooth Mesh Protocol v1.1 specification
+ */
+
+#include <zephyr/kernel.h>
+#include <string.h>
+#include <errno.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <zephyr/sys/atomic.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/sys/byteorder.h>
+
+#include <zephyr/net/buf.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/mesh.h>
+
+#include "mesh.h"
+#include "net.h"
+#include "subnet.h"
+#include "settings.h"
+#include "brg_cfg.h"
```

Comment:
```suggestion
#include "brg_tbl.h"
```

Reply:
I don't think so, because this module includes two settings - brige enable, and bridging table.